tests/selenium/base.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, cap=None):
 
     def read_ini(self):
         self.modules = []
+        self.servers = 1
```

Comment:
This must come from PHP:
```
count($this->get('imap_servers')) + count($this->get('feeds'))
```

Reply:
Testing scripts set up the servers themselves - either via config or via the UI to test the UI. So, you should be aware of the target server count at each moment of testing. No need to get it from PHP, I think.